### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -590,6 +591,7 @@ In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the
 ## INPUTS
 
 ## NOTES
+Note: The the mentioned changes in the above sections including changes under "-Identity" and "-Filter" paramters are not applicable for customers and tenants that are or have been enabled for Regionally Hosted Meetings for Skype for Business Online.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:594]:
```suggestion
The changes mentioned in these sections, including changes under "-Identity" and "-Filter" parameters, are not applicable to customers and tenants that are or have previously been enabled for Regionally Hosted Meetings for Skype for Business Online. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes if you agree with them. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward @dariomws @get-itips

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -245,9 +244,10 @@ In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attrib
 
 In the Teams PowerShell Module version 3.0.0 or later, the format of the AssignedPlan and ProvisionedPlan attributes has changed from XML to JSON array. Previous XML filters (For example, `-Filter "AssignedPlan -eq '<some-xml-string>'"`) will no longer work. Instead, you need to update your filters to use one of the following formats:
 
-- All users with an AssignedPlan that starts with MCO: `-Filter "AssignedPlan -eq 'MCO*'"`
-- All users with an AssignedPlan that contains MCO: `-Filter "AssignedPlan -eq '*MCO*'"`
-- All users with an AssignedPlan that ends with "MCO": `-Filter "AssignedPlan -eq '*MCO'"`
+- All users with an AssignedPlan matches MCO: `-Filter "AssignedPlan -eq 'MCO'"`
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:247]:
```suggestion
- All users with an AssignedPlan that matches MCO: `-Filter "AssignedPlan -eq 'MCO'"`
```

**Comment by yogkumgit** [comment]
@isrumnon

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -232,7 +232,6 @@ In the Teams PowerShell Module version 3.0.0 or later, filtering functionality i
 - UserDirSyncEnabled
 - TeamsWorkLoadPolicy
 - VoiceRoutingPolicy
-- LastName (available in Teams PowerShell Module 4.2.0 and later)
```
Comment (pbafna03) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:235]: @isrumnon  - is this not there in 4.2.1? if yes, let's add that instead of full removal

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**